indexer/services/roundtable/src/tasks/update-compliance-data.ts

```diff
@@ -165,6 +176,12 @@ export default async function runTask(
       undefined,
       { provider: complianceProvider.provider },
     );
+    stats.gauge(
+      `${config.SERVICE_NAME}.${taskName}.num_inactive_addresses_with_stale_compliance`,
+      inactiveAddressesWithStaleCompliance,
+      undefined,
```

Comment:
what is this undefined?

Reply:
it is the sample rate. For the other count metrics, we used undefined so i kept it the same for this count metric.

Reply:
Let's use `STATS_NO_SAMPLING` this constant